File: Server/src/main/java/com/api/server/foodb/PdbIdentifier.java
package com.api.server.foodb;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import jakarta.validation.constraints.Size;
import lombok.Getter;
import lombok.Setter;

import java.time.Instant;

@Getter
@Setter
@Entity
@Table(name = "pdb_identifiers")
public class PdbIdentifier {
    @Id
    @Column(name = "id", nullable = false)
    private Integer id;

    @Column(name = "compound_id")
    private Integer compoundId;

    @Size(max = 255)
    @Column(name = "pdb_id")
    private String pdbId;

    @Column(name = "created_at")
    private Instant createdAt;

    @Column(name = "updated_at")
    private Instant updatedAt;

}

File: DotNetServer/DotNetServer/Entities/Foodb/PdbIdentifier.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore;

namespace DotNetServer.Entities.Foodb;

[Table("pdb_identifiers")]
[Index("CompoundId", Name = "index_pdb_identifiers_on_compound_id")]
public partial class PdbIdentifier
{
    [Key] [Column("id")] public int Id { get; set; }

    [Column("compound_id")] public int? CompoundId { get; set; }

    [Column("pdb_id")] [StringLength(255)] public string? PdbId { get; set; }

    [Column("created_at", TypeName = "datetime")]
    public DateTime? CreatedAt { get; set; }

    [Column("updated_at", TypeName = "datetime")]
    public DateTime? UpdatedAt { get; set; }
}